src/bionics.cpp

```diff
@@ -66,9 +66,13 @@ const efftype_id effect_took_xanax( "took_xanax" );
 const efftype_id effect_visuals( "visuals" );
 const efftype_id effect_weed_high( "weed_high" );
 
-static const trait_id trait_HYPEROPIC( "HYPEROPIC" );
-static const trait_id trait_MYOPIC( "MYOPIC" );
 static const trait_id trait_PROF_MED( "PROF_MED" );
```

Comment:
These constants aren't used anywhere in bionics.cpp, so removing them shouldn't do any harm.